.github/workflows/pr-repair.yml

```
@@ -419,6 +501,60 @@ jobs:
+      # nothing happened here supports. Every reply names the commit, so
+      # reopening is one click if the answer is not good enough.
+      #
+      # `continue-on-error`: a repair that landed and verified must not be
+      # reported as failed because the review API was briefly unavailable.
+      - name: Reply to each finding this repair actually touched, then resolve it
+        if: steps.token.outputs.have == 'true' && steps.ship.outcome == 'success' && steps.threads.outputs.count != '0' && steps.threads.outputs.count != ''
+        continue-on-error: true
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_REPO: ${{ github.repository }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:18:27: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)